doc/whats_new/v0.23.rst

```diff
@@ -73,6 +73,11 @@ Changelog
   `store_cv_values` is `True`.
   :pr:`15652` by :user:`Jérôme Dockès <jeromedockes>`.
 
+- |Fix| Fixed a bug in :class:`linear_model.RidgeRegressionCV` to pass a
```

Comment:
Shouldn't this be `RidgeClassifierCV`?

Also I had fixed a bug, which raises error when `RidgeClassifierCV` is fed with classifier scoring and CV!=None

Reply:
I fixed this issue. I don't think that we have to go into details regarding `cv` since we had a bug in both cases (`cv== None` or `cv != None`). The fix might be different for both case but no need to explain in what's new.

Reply:
okay got it. Thanks for the info.